qa/suites/upgrade-clients/client-upgrade-nautilus-pacific/nautilus-client-x/rbd/3-workload/rbd_notification_tests.yaml

```diff
@@ -1,14 +1,14 @@
 tasks:
 - parallel:
   - workunit:
-      branch: nautilus
+      branch: pacific
```

Comment:
I can't understand why these changes are here?

Reply:
@trociny the src/test/librbd/test_notify.py script is picked from branch specified in these workunits, in order to pick : https://github.com/ceph/ceph/pull/41514/files#diff-ac8940c0fdcd92dc4a7129f21a53e256a84e9d8245710d18cd0bd0e197b69e43R148 we can either backport it to stable branches or keep picking this script from master as well, since nothing has changed in the script from nautilus.

But since in future we might want to get rid of this check, I saw backport option to branches better.

Reply:
I would prefer if we backported the change to the script to all necessary stable branches.

Reply:
will do, spilted the change and added commit about it, thanks!